### PR TITLE
8.0 - Renaming of configuration files

### PR DESCRIPTION
**Pull request by aab-odoo**
openerp-server.conf is renamed into odoo.conf.
openerp_serverrc is renamed into odoorc.

There is a fallback so that the renaming doesn't break scripts of
users that already have a custom openerp_serverrc.

Also print the configuration file used (if any) in the logs.

### CODE REVIEW

File: doc/setup/install.rst [hunk 1]
```diff
@@ -137,6 +137,11 @@ When the configuration file is edited, Odoo must be restarted using
     $ sudo service odoo restart
     Restarting odoo: ok
 
+.. warning:: if the configuration file :file:`/etc/odoo/openerp-server.conf`
```
Comment (antonylesuisse) [inline doc/setup/install.rst]: Why ? It should be the opposite.

Comment (aab-odoo) [inline doc/setup/install.rst]: We don't want to break scripts of people already having a custom configuration in /etc/odoo/openerp-server.conf. With this fallback, odoo will continue using the configuration in this file, unless the user deletes it.

**Comment by xmo-odoo** [comment]
I also think it'd make more sense to use the new name by default and fallback onto the old one, that allows a more seamless upgrading. Also maybe trigger a warning when falling back on the old name.

**Comment by odony** [comment]
Do we really want to do this in 8.0? It is very similar to a change of API, and in that context I understand @aab-odoo's concern of being really conservative when multiple config files exist. Shouldn't this change be retargeted at master along with the other higher-risk changes? The config file doesn't strike me as "user-facing" text, but rather "sysadmin-facing" tech stuff.

That said, if we really do it in 8.0, we could use the new name by default, fallback to the old one, and crash at startup if both config files are present (quite unlikely), making it obvious that a manual operation is necessary.

But again, this change is bound to bite *someone* at *some point* regardless of how we do it, so it'd be safer in master, where people expect surprises.

**Comment by xmo-odoo** [comment]
@odony I'm fine with only doing that as part of the "technical" rename in master (with a fallback either way)

**Comment by antonylesuisse** [comment]
We dont change v8. This branch  should target master and order should be odoorc then openerp.

**Comment by rim-odoo** [comment]
@aab-odoo Is this PR still valid or can we close it?

**Comment by aab-odoo** [comment]
It is valid but should be retargeted to master and slightly adapted.

**Comment by rim-odoo** [comment]
@aab-odoo Can you do it? Is there a task somewhere?

**Comment by aab-odoo** [comment]
There is probably no task for it. I unfortunately don't have time for this right now. After the Odoo days maybe. No problem if someone else takes it over.

**Comment by antonylesuisse** [comment]
Let's try not to miss v10 :)

**Comment by sle-odoo** [comment]
closing with c59eb4ffc1da64333075af9d89a620cb9a037cae and 630b4b1d556b84649cf7b9bfe7a170114d918a36

**Pull request closed without merging**